flink-table/flink-sql-parser/src/test/java/org/apache/flink/sql/parser/MaterializedTableStatementParserTest.java

```diff
@@ -358,6 +361,15 @@ void testDropTemporaryMaterializedTable() {
         sql(sql).fails("DROP TEMPORARY MATERIALIZED TABLE is not supported.");
     }
 
+    @ParameterizedTest(name = "{index}: {0}")
+    @MethodSource("inputForCreateOrAlterMaterializedTable")
+    void testCreateOrAlterMaterializedTable(Map.Entry<String, String> sqlToExpected) {
+        final String sql = sqlToExpected.getKey();
+        final String expected = sqlToExpected.getValue();
+
+        sql(sql).ok(expected);
+    }
```

Comment:
what is the difference between this test body and `testCreateMaterializedTable`?
why can't it be the same test?